kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -49,18 +48,9 @@ trait ProcBuilder {
 
   protected def env: Map[String, String] = conf.getEnvs
 
-  protected val workingDir: Path
-
-  final lazy val processBuilder: ProcessBuilder = {
-    val pb = new ProcessBuilder(commands: _*)
+  protected def launcher: SparkLauncher
```

Comment:
can SparkLauncher be able to launch the coming Flink SQL eninge?

Reply:
Probably not.